proseco/guide.py

```diff
@@ -134,6 +134,9 @@ def run_search_stages(self):
         """
         cand_guides = self.meta['cand_guides']
         self.log("Starting search stages")
+        if len(cand_guides) == 0:
```

Comment:
@taldcroft I looked at adding similar tweaks to what you used in proseco.acq, but for the empty candidate list, it seemed just as reasonable to stop running the checks and return the empty table of candidates.  Does this make sense?

As far as I could tell, not *selecting* any stars later through the search stages resulted in just the all-False mask within cand_guides, which returned a 0-list table with an 'id' column defined... which seemed all good.